test/system/roles/admin/dashboard/dashboard_verifier.rb

```diff
@@ -8,7 +8,7 @@ class AdminDashboardVerifier < ApplicationSystemTestCase
   @@system_test_utils = SystemTestUtils.new(nil)
 
   def verify_user(user, should_exist: true)
-    Capybara.using_wait_time(4) do
+    Capybara.using_wait_time(8) do
```

Comment:
I had a feeling this needed to be raised, though it surprises me we don't need to have wait times like this in a lot of the other system tests.